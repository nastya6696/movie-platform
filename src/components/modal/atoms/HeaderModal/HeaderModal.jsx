import styles from './styles.module.scss';

export const HeaderModal = ({title}) => (
  <h2 className={styles.title}>{title}</h2>
)
