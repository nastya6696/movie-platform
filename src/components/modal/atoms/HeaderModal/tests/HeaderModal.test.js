import ShallowRenderer from "react-test-renderer/shallow";
import React from "react";
import {HeaderModal} from "../HeaderModal";

test('should create simple HeaderModal component', () => {
  const component = new ShallowRenderer().render(
    <HeaderModal title='Test modal header'/>
  );
  expect(component).toMatchSnapshot();
})
