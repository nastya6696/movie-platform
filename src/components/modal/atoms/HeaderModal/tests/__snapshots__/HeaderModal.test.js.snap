// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should create simple HeaderModal component 1`] = `
<h2
  className="title"
>
  Test modal header
</h2>
`;
